mmv1/products/orgpolicy/api.yaml

```
@@ -0,0 +1,99 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: OrgPolicy
+display_name: Organization Policy
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://orgpolicy.googleapis.com/v2/
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Organization Policy API 
+    url: https://console.cloud.google.com/apis/api/orgpolicy.googleapis.com/overview
+objects:
+  - !ruby/object:Api::Resource
+    name: 'CustomConstraint'
+    create_url: '{{parent}}/customConstraints'
+    self_link: '{{parent}}/customConstraints/{{name}}'
+    base_url: '{{parent}}/customConstraints/{{name}}'
+    update_verb: :PATCH
+    min_version: beta
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/resource-manager/docs/organization-policy/creating-managing-custom-constraints'
+        'Supported Services':
+          'https://cloud.google.com/resource-manager/docs/organization-policy/custom-constraint-supported-services'
+      api: 'https://cloud.google.com/resource-manager/docs/reference/orgpolicy/rest/v2/organizations.constraints'
+    description: |
+      Custom constraints are created by administrators to provide more granular and customizable control over the specific fields that are restricted by your organization policies.
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: parent
+        description: |
+          The parent of the resource.
```

Let's call out the format explicitly here:

```suggestion
          The parent of the resource, an organization. Format: `"organizations/{{organization}}"`.
```

We can fix the docs if this ever becomes not true (and there's no code stopping it from working w/o the fix)

---

Agreed, thanks for catching these